package.json

```diff
@@ -2,13 +2,26 @@
   "name": "orbit-db-cache",
   "version": "0.2.4",
   "description": "Local cache for orbit-db",
+  "keywords": [
+    "orbit",
+    "orbitdb",
+    "cache"
+  ],
+  "homepage": "https://github.com/orbitdb/orbit-db-cache",
+  "bugs": "https://github.com/orbitdb/orbit-db-cache/issues",
   "main": "index-nodejs.js",
   "browser": "index-browser.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "lint": "standard --env=mocha"
   },
   "author": "Haad",
+  "contributors": [
+    "RichardLitt",
+    "mistakia",
+    "thiagodelgado111",
+    "adam-palazzo"
```

Comment:
I think these ought to be alphabetical? There's a tool in Atom to automatically do that.

Reply:
Alphabetical instead of in order of contribution?

Reply:
Yeah. I don't think order of contribution is clear.